### ISSUE TITLE
v6 pro faThumbtack not working

### ISSUE THREAD

**Issue by mmaloon** [FortAwesome/Font-Awesome#18661]
**Describe the bug**
after upgrading to @fortawesome 6.0.0, the faThumbtack icon is broken

**To Reproduce**
Not sure how to create a reproducible example with the "pro" icons, but the issue is pretty clear from looking at the code in @fortawesome/pro-solid-svg-icons/faThumback.js. It is `require`ing itself and then exporting as `faThumbTack` instead of `faThumbtack`.

@fortawesome/pro-solid-svg-icons/faThumback.js:
```javascript
'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
var source = require('./faThumbtack'); // <-- requiring self??
exports.definition = {
  prefix: source.prefix,
  iconName: source.iconName,
  icon: [
    source.width,
    source.height,
    source.aliases,
    source.unicode,
    source.svgPathData
  ]};

exports.faThumbTack = exports.definition; // <-- should be faThumbtack
exports.prefix = source.prefix;
exports.iconName = source.iconName;
exports.width = source.width;
exports.height = source.height;
exports.ligatures = source.aliases;
exports.unicode = source.unicode;
exports.svgPathData = source.svgPathData;
exports.aliases = source.aliases;
```

**Expected behavior**
faThumbtack should work like all other icons.

**Version and implementation**
Version: @fortawesome/pro-solid-svg-icons 6.0.0

- [x] SVG with JS
- [ ] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

@robmadole could you please take a look?

**Comment by MohammadMRD** [comment]
Hello,
I'm having a similar issue with `faPaintbrush`

```js
import { faPaintbrush } from '@fortawesome/free-solid-svg-icons/faPaintbrush';
```

**Comment by tagliala** [comment]
@MohammadMRD thanks for the report. I've created a separate issue at #18724

**Comment by 1st8** [comment]
Same issue with pro-duotone-svg-icons/faThumbtack

**Comment by rbare1** [comment]
I haven't tested it since the official release, but this seems to be the same as the issue in the beta [#18342](https://github.com/FortAwesome/Font-Awesome/issues/18342)

**Comment by robmadole** [comment]
All, this should be resolved now in the freshly released `6.1.0`. Please try it out and let me know.

**Comment by 1st8** [comment]
Confirm resolved in 6.1.0

Cheers!
